src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -70,6 +95,7 @@ export default function CustomFieldInputText(props) {
         aria-haspopup={props.ariaProps.haspopup}
         defaultValue={props.defaultValue}
         className={styles.input}
+        style={{ '--numIcon': numIcons() }}
```

Comment:
Very cool usage of a local style variable! See the above comment about a possible refactor of this behavior, though.

An alternate option to the previous suggestion would be to compute the padding here, to keep the CSS as simple as possible.

Reply:
I think it is nice making use of the css variables